lib/squadster/workers/notify_attendants.ex

```diff
@@ -36,7 +36,8 @@ defmodule Squadster.Workers.NotifyAttendants do
     """
       {
         "text": "#{gettext("You are on duty tomorrow!")}",
-        "target": #{user.id}
+        "target": #{user.id},
+        "type": #{user.message_type}
```

Comment:
This should be an array `"types": ["text", "voice"]`

Reply:
:+1: